repos.d/deb/deepin.yaml

```diff
@@ -26,3 +26,30 @@
     - desc: Deepin home
       url: https://www.deepin.org/
   groups: [ all, production, deepin ]
+
+- name: deepin_23
+  type: repository
+  desc: deepin 23
+  statsgroup: Debian+derivs
+  family: debuntu
+  ruleset: [ debuntu, deepin ]
+  color: '949393'
+  minpackages: 28000
```

Comment:
It only happens to have 9009 packages (is this expected?), minpackages needs to be fixed to 8000 or so.

Reply:
Fixed to 8000 now
cc @UTsweetyfish is this expected to have 9009 packages?

Reply:
\* I'm asking because deepin 20 had much more (29k) packages.

Reply:
deepin 20 is based on debain, but deepin 23 is an independent distribution. So this change is possible